web/views.py

```
@@ -6949,3 +6951,170 @@ def delete_post(request, post_id):
     if request.method == "POST":
         post.delete()
     return redirect("social_media_dashboard")
+
+
+# infographics
+
+
+def infographics(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `infographics`

(ANN201)